src/auro-datepicker.js

```diff
@@ -69,13 +73,21 @@ export class AuroDatePicker extends LitElement {
   constructor() {
     super();
 
+    /**
+     * @private
+     */
+    this.util = new AuroDatepickerUtilities();
```

Comment:
Consider grouping all your private defaults. See auro-icon as a good example. 

https://github.com/AlaskaAirlines/auro-icon/blob/6dcf1eb795cc7e032d713f050504eb51fc82c5b6/src/auro-icon.js#L52

Reply:
I see your point. I took a slightly different approach - I put all the utility declarations first as I see them as something notably different than the other properties.